tests/layer_tests/common/layer_test_class.py

```diff
@@ -93,9 +93,9 @@ def _test(self, framework_model, ref_net, ie_device, precision, ir_version, temp
         #     assert flag, '\n'.join(resp)
 
         config = None
-        # GPU default execution precision is FP16, so if we want to check FP32 inference
+        # GPU and some CPU arm platforms default execution precision is FP16, so if we want to check FP32 inference
         # we need to set explicit precision hint
-        if ie_device == 'GPU' and precision == 'FP32':
+        if precision == 'FP32':
```

Comment:
I think we test the default scenario in the layer test that is more important.

Reply:
Is it possible to figure out why the accuracy is degraded for the default scenario on these plugins? This fix can just hide the issue in the plugins.

Reply:
hi, @rkazants :  because CPU Plugin use fp16 inference precision by default for some specific arm platform(macOS) cases. for these cases, Nodes will be converted to fp16 precision during runtime. detail information can be found in the ticket-122182

Reply:
We should test the default scenario and no need to change inference precision. If you need to top up threshold, it is possible to configure in the test.

Reply:
Agree with Roman, it is better to test default scenario. GPU+FP16 and ARM+FP16. It will uncover problems in FP16 inference which we definitely have currently

Reply:
yes, agree that we can try to top up threshold for these kinds of cases to solve this issue. 
But does it seems functional mismatch if the layer tests really run in fp16 precision(e.g.  we don't set {''INFERENCE_PRECISION_HINT': 'f32'} for some ARM platforms) while  layer tests script [set precision to fp32](https://github.com/openvinotoolkit/openvino/blob/bd57987c36f71a0f47e1dc3755a6bb7a706c12dd/tests/layer_tests/README.md?plain=1#L28)? I mean dose this [precision](https://github.com/openvinotoolkit/openvino/blob/bd57987c36f71a0f47e1dc3755a6bb7a706c12dd/tests/layer_tests/README.md?plain=1#L28) stands for the target test precision?

Reply:
I think it should be a question to plugins, why we have some deconvergence between plugins so that different precision is used by default.
Please test default scenario and set separate thresholds for ARM. Otherwise, some issue can e hidden in the plugin

Reply:
have changed to keep test default scenario and set separate thresholds(former fp16 threshold) for ARM CNN model cases to align with CPU Plugin